src/index.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable comma-dangle */
```

Comment:
Why disable comma-dangle and delete the comma here? I'm sure you had a reason to do it, but I thought I'd ask about it since this seems strange

Reply:
I would like to leave this for ow.  Can you add an issue if you want?